charts/pega/templates/pega-environment-config.yaml

```diff
@@ -172,6 +172,15 @@ data:
 {{ if .Values.stream.enabled }}
   # Whether to enable connecting to an external stream cluster.  "true" for enabled, "false for disabled"
   EXTERNAL_STREAM: "true"
+  {{- if .Values.stream.provider }}
+  SERVICES_STREAM_PROVIDER: "{{ .Values.stream.provider }}"
+  {{- end }}
+  {{- if .Values.stream.clusterName }}
+  KAFKA_CLUSTER_NAME: "{{ .Values.stream.clusterName }}"
+  {{- end }}
+  {{- if .Values.stream.validate }}
+  SERVICES_STREAM_VALIDATE: "{{ .Values.stream.validate }}"
```

Comment:
Where does this param used? I do not find its reference in prweb.xml PR?

Reply:
[Here](https://github.com/pegasystems/docker-pega-web-ready/blob/master/tomcat-conf/Catalina/localhost/prweb.xml#L113). In case of CMC, it is defaulted to `true` when External stream is set to true. 

However, to leverage the configuration, the parameter is exposed for other Cloud3.x Deployments.